src/overlays/actors/ovl_En_Toryo/z_en_toryo.h

```diff
@@ -6,9 +6,27 @@
 
 struct EnToryo;
 
+typedef void (*EnToryoActionFunc)(struct EnToryo* this, GlobalContext* globalCtx);
+
+typedef struct {
+    AnimationHeader* anim;
+    f32 unk_4;
+    u8 mode;
+    f32 transitionRate;
+} EnToryoAnimation;
```

Comment:
if this is only used internally in the c file then you can define it in there instead of in the header.
and if you could, put offset comments and a size comment